src/setup/serverless-config.go

```diff
@@ -0,0 +1,28 @@
+package setup
+
+type Serverless struct {
+	// TODO: add serverless.yml fields
+}
+
+// AddFunctionConfig Adds a function to the service. If parallelism = n, then it defines n functions. Also deploys all producer-consumer subfunctions.
+func (s *Serverless) AddFunctionConfig() {
+	// TODO: implement function
+}
+
+// CreateServerlessConfigFile dumps the contents of the Serverless struct into a yml file.
+func (s *Serverless) CreateServerlessConfigFile() {
+	// TODO: implement function
+}
+
+// RemoveService removes the service defined in serverless.yml
+func RemoveService() string {
+	// TODO: implement function
+	return ""
+}
+
+// Deploys the functions defined in the serverless.com file
+
+func DeployService() string {
```

Comment:
rename to DeployServices

Reply:
In serverless.com framework the word service refers to the Framework's unit of organization. It refres to a set of functions defined in a single serverless.yml file. https://www.serverless.com/framework/docs/providers/aws/guide/services. If we have a single serverless.yml file, we deploy a single service? Are you sure we want to rename it?